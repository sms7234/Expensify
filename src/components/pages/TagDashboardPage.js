import React from 'react';
import TagList from '../lists/TagList';
import TagSummary from '../independents/TagSummary';
import TagListFilters from '../filters/TagListFilters';

const TagDashboardPage = () => (
  <div>
    <TagSummary />
    <TagListFilters />
    <TagList />
  </div>
)

export default TagDashboardPage;
