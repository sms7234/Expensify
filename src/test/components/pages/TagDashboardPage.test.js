import React from 'react';
import {shallow} from 'enzyme';
import TagDashboardPage from '../../../components/pages/TagDashboardPage';

test('should render the dashboard page', () => {
  const wrapper = shallow(<TagDashboardPage />);
  expect(wrapper).toMatchSnapshot();
})
