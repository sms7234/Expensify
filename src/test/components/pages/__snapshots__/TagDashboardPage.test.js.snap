// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the dashboard page 1`] = `
<div>
  <Connect(TagSummary) />
  <Connect(TagListFilters) />
  <Connect(TagList) />
</div>
`;
